spec/features/menu_items/wiki_menu_item_spec.rb

```diff
@@ -92,8 +92,8 @@
     visit project_wiki_path(project, wiki_page)
 
     # creating the menu item with the pages name for the menu item
-    click_link_or_button "More"
-    click_link_or_button "Configure menu item"
+    page.find('[data-test-selector="wiki-more-dropdown-menu"]').click
```

Comment:
You can use our test helpers for things like that:
`page.find_test_selector("foo-bar").click`